wordpress-mobile/WordPress-iOS
# Aztec dark mode

Refs #12320. This PR implements dark mode for Aztec.

![Simulator Screen Shot - iPhone Xs - 2019-08-22 at 22 24 06](https://user-images.githubusercontent.com/4780/63550904-9379a800-c52b-11e9-8f88-973e772e6695.png)

There are a few small changes in light mode due to using the semantic or system colors now. Please let me know if this isn't acceptable!

![editor-1](https://user-images.githubusercontent.com/4780/63549011-74791700-c527-11e9-9a59-2fe11af2e7ad.png)

**To test:**

* Build and run in iOS 13 with dark mode enabled. Check the UI in the editor looks good and matches the screenshot above.
* Run in light mode on iOS 13 and check everything looks correct.
* Run on iOS 12 and check everything looks correct.

Update release notes:
